### ISSUE TITLE
Build Landing Page

### ISSUE THREAD

**Issue by ashatat** [ashatat/e-commerce#4]

**Comment by amusameh** [comment]
@RamyAlshurafa  @FarahZaqout  @ashatat  @ishak52 
I think we should make a page layout or main layout for the elements the exists in each page (Header, Footer, SideNavbar ) and call it `main layout` instead of repeating it in every page.
Should we make thais a sperate Epic or an issue?

**Comment by ashatat** [comment]
@amusameh   yeah I agree, definitely, we should have a main layout epic

**Comment by FarahZaqout** [comment]
Yes. Also, @ashatat This is some really good detailing. We should copy paste it in the detailed issues that are related to this and then after that remove  them from the epic as we agreed last meeting.

Great work man. Holy shit!
